.travis.yml

```diff
@@ -1,109 +1,110 @@
-
+  global:
+    - REPO_DIR=pystan
+    - BUILD_COMMIT=v2.19.1.1
+    - BUILD_DEPENDS=Cython
+    - TEST_DEPENDS=
+    - MB_ML_VER=2014
+    - DOCKER_TEST_IMAGE=multibuild/xenial_x86_64
+    - PLAT=x86_64
+    - UNICODE_WIDTH=32
+    - NP_BUILD_DEP=1.8.2
+    - WHEELHOUSE_UPLOADER_USERNAME=pystan-worker
+    - WHEELHOUSE_UPLOADER_REGION=IAD
+    # Following generated with
+    # travis encrypt -r stan-dev/pystan-wheels WHEELHOUSE_UPLOADER_SECRET=<the api key>
+    - secure: "RPK/AsxmDNiiJ5WZwRr8oN5Ybj+/IaCNp/zF8ZwOO4XXG0RF56n1fGeqW+tfHhsOjjwBZtYmQSqSDrcd3cQw564yoBpbTy1YIFPMVP5gELFE4xw9A7KGOcGJ22RMNbqAq4esl8YqOLVjLsFQbx0aCrcRripnj0tjK/6nOemYt6+Oo8rm2SZjRnclmE7UsVSoYYll/cE+H/wTLZo3nHnRl6JZUcTa5dQ6i/GUR+kMzA7u4yGI6n16THeBIVI6dAOgabmInahZad6v7EwUWGZrwUHBmQmZhqMkPU9YVEd3I4MUgJ/9acl4ezlQuEZ4CfagSKqWMUpa1nqjsuNf374wZRi3jv4urTd6ouk+32TkFBQDHrBpdSCRgzn1Vp9Pogs4sThIgUOSEDx90nS9lhmQ5EFm8fIz2W4h1Relo0ra+FKvahIBpMCSW20Ozty4GiyY4bIYt9vzSZCblp5HlxWF5UW1zRIExtJvSipSEHElk8SvFryunFacqt2T36OeaYhp2mA5LGqTcNWMTTNsM4Z0Vw7CzeB9KOgTT1FRKVjM9oiVO9blAj0TkseBYnR0PuQ6EdHmNq8bO+oupaafca/cq6j7GRiv5n7pL66LV75xv+8gQ4VpLY6Z31+DRZ74eh/nobsIJobYz6H5N8wjhuGZAtMHIeisZrdjWLOGiqGa9KY="
 
 language: python
 cache:
   - pip
 python: 3.5
-dist: trusty
+dist: xenial
 services: docker
 
-matrix:
-  exclude:
-      # Exclude the default Python 3.5 build
-      - python: 3.5
-  include:
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=2.7
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=2.7
-        - UNICODE_WIDTH=16
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=3.5
-        - NP_BUILD_DEP="1.9.3"
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=3.6
-        - NP_BUILD_DEP="1.9.3"
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=3.7
-        - NP_BUILD_DEP=1.14.5
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=3.8
-        - NP_BUILD_DEP=1.14.5
-    - os: osx
-      language: objective-c
-      env:
-        - MB_PYTHON_VERSION=2.7
-    - os: osx
-      language: objective-c
-      env:
-        - MB_PYTHON_VERSION=3.5
-        - NP_BUILD_DEP="1.9.3"
-    - os: osx
-      language: objective-c
-      env:
-        - MB_PYTHON_VERSION=3.6
-        - NP_BUILD_DEP="1.9.3"
-    - os: osx
-      language: generic
-      env:
-        - MB_PYTHON_VERSION=3.7
-        - NP_BUILD_DEP=1.14.5
-    - os: osx
-      language: generic
-      env:
-        - MB_PYTHON_VERSION=3.8
-        - NP_BUILD_DEP=1.14.5
+jobs:
+  - os: linux
+    env:
+      - MB_PYTHON_VERSION=2.7
+  - os: linux
+    env:
+      - MB_PYTHON_VERSION=2.7
+      - UNICODE_WIDTH=16
```

Comment:
Latest manylinux2014 has dropped support for 2.7. Not sure about manylinux1.

And 2.7 is eol, and 2.19.1 wheel is in pypi.

Reply:
Let's get rid of it then.

Reply:
Do you want me to add a commit for it, or do you want to do it yourself?